src/Types/Encoder.cs

```diff
@@ -61,9 +61,9 @@ class Serializer
 
 #if NETMF
         // NETMF DateTime ticks origin is 1601/1/1
-        const long epochTicks = 116444736000000000; // 1970-1-1 00:00:00 UTC
+        const long epochTicks = 621355968000000000; // 1601-1-1 00:00:00 UTC
 #else
-        const long epochTicks = 621355968000000000; // 1970-1-1 00:00:00 UTC
+        const long epochTicks = 116444736000000000; // 1970-1-1 00:00:00 UTC
 #endif
```

Comment:
Only the epoch on NETMF is different from all other platforms. The NANO framework share the same AMQP code as NETMF but its runtime has changed the epoch to be the same as other .Net frameworks.
So we can fix the bug by changing line 62 to `#if NETMF && !NANOFRAMEWORK_1_0` and keeping the values as is.

Reply:
OK - Understood 

I thought that if the EpochTicks origin is 1601, then the epochTicks would be the higher value of 621355968000000000 rather than 116444736000000000?